nautobot/extras/models/relationships.py

```diff
@@ -1,5 +1,4 @@
 import logging
-from collections import OrderedDict
```

Comment:
Use of `OrderedDict` instead of simple `dicts` made testing of `get_relationships_data()` more difficult; we're not using any of the advanced features of `OrderedDict` here, we just want consistent ordering of keys, which is inherent to `dict` in Python 3.7 and later.